service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -0,0 +1,108 @@
++
+[source,terminal]
+----
+$ oc get servicemeshcontrolplanes.v1.maistra.io <smcp_name> -o yaml > <smcp_name>.v1.yaml
+----
++
+. Generate and retrieve a sample v2 `ServiceMeshControlPlane` that contains information about your configuration. You can modify this version of your `ServiceMeshControlPlane` as a starting point.
++
+[source,terminal]
+----
+$ oc get smcp <smcp_name>.v2.yaml
+----
++
+. Create a project. In the {product-title} console Project menu, click `New Project` and enter a name for your project, `istio-system-upgrade`, for example. Or, you can run this command from the CLI.
++
+[source,terminal]
+----
+$ oc new-project istio-system-upgrade
+----
++
+. Create a `ServiceMeshControlPlane` in the new namespace. On the command line, run the following command to deploy the control plane with the v2 version of the `ServiceMeshControlPlane` that you retrieved. In this example, replace `<V2_SMCP> `with the path to your file.
++
+[source,terminal]
+----
+$ oc create -n istio-system-update -f <V2_SMCP>.yaml
+----
++
+Alternatively, you can use the console to create the control plane. In the console, click *Project*. Then, select the project name you just entered.
++
+..Click *Operators* -> *Installed Operators*.
+..Click *Create ServiceMeshControlPlane*.
+..Select *YAML view* and paste the following example into the field.
++
+..Click *Create*.
+
+[id="ossm-upgrading-differences_{context}"]
+== Configuring the 2.0 `ServiceMeshControlPlane`
+
+The `ServiceMeshControlPlane` resource has been updated for {ProductName} version 2.0. Next, modify the v2 `ServiceMeshControlPlane` resource to take advantage of the new features and to fit your deployment. Consider the following changes to the specification and behavior of {ProductName} 2.0 as you're modifying your `ServiceMeshControlPlane` resource. You can also refer to the {ProductName} 2.0 product documentation for updates to features you use.
+
+[id="ossm-upgrading-differences-arch_{context}"]
+=== Architecture changes
+
+The architectural units used by previous versions have been replaced by istiod. It that replaces and combines the functions of Galley, Citadel, Sidecar Injector, and Mixer telemetry and policy deployments.
+
+Mixer policy and telemetry plugins are now supported through WASM extensions in istiod. However, Mixer can be enabled for policy and telemetry if you need to integrate legacy Mixer plugins.
+
+Security Discovery Service (SDS) is used to distribute certificates and keys to sidecars directly from istiod. In {ProductName} version 1.1, secrets were generated by Citadel, which were used by the proxies to retrieve their client certificates and keys.
+
+[id="ossm-upgrading-differences-config_{context}"]
+=== Configuration changes
+
+The `ServiceMeshControlPlane` has been upgraded to version 2. The default resource version for `ServiceMeshControlPlane` resources. The v2 resource must be used for {ProductName} 2.0 installations. 
```

Comment:
I feel like there's either a word missing here (this is the default) or that there should be a comma not a period after that first sentence?